tardis/analysis.py

```diff
@@ -37,9 +40,12 @@ def __init__(
         self.last_line_interaction_shell_id = last_line_interaction_shell_id[
             mask
         ]
-        self.last_line_interaction_angstrom = output_nu.to(
-            u.Angstrom, equivalencies=u.spectral()
-        )[mask]
+        self.last_line_interaction_out_angstrom = u.Quantity(
+            output_nu[mask], "Hz"
+        ).to(u.Angstrom, equivalencies=u.spectral())
+        self.last_line_interaction_in_angstrom = u.Quantity(
```

Comment:
It might be better to rename output_nu and last_interaction_in_nu so that they are consistent. 
e.g. output_nu and input_nu